charts/aws-ebs-csi-driver/templates/node.yaml

```diff
@@ -103,7 +104,7 @@ spec:
               mountPath: /csi
             - name: registration-dir
               mountPath: /registration
-          {{- with .Values.resources }}
+          {{- with .Values.node.resources }}
```

Comment:
Similarly here. I understand this makes more sense, but we still have to make sure we don't break existing setups without any warning.